core/tm/routes/profesion.ts

```diff
@@ -14,6 +14,14 @@ router.get('/profesiones/:id*?', (req, res, next) => {
             res.json(data);
         });
 
+    } else if (req.query.codigo) {
+        profesion.find({ codigo: req.query.codigo }, (error, data) => {
```

Comment:
Era un llamado de atención a seguir las guías de desarrollo. 

```suggestion
        const profesion = await profesion.find({ codigo: req.query.codigo });
```

Acomodar de ser posible todo el archivo, son dos minutos